source/MaterialXRenderGlsl/GlslProgram.cpp

```diff
@@ -540,11 +540,13 @@ bool GlslProgram::bindTexture(unsigned int uniformType, int uniformLocation, con
 
         if (haveImage)
         {
-            int textureLocation = imageHandler->getBoundTextureLocation(imageDesc.resourceId);
-            if (textureLocation < 0) return false;
```

Comment:
You don't want to return false if the textureLocation is negative?

Reply:
It will later on. Wanted to do the GL check as well before returning.

Reply:
Ah, ok